src/main/java/mate/academy/Main.java

```diff
@@ -1,7 +1,50 @@
 package mate.academy;
 
+import java.math.BigDecimal;
+import java.util.List;
+import java.util.Optional;
+import mate.academy.dao.BookDao;
+import mate.academy.lib.Injector;
+import mate.academy.model.Book;
+
 public class Main {
+    private static final Injector INJECTOR = Injector.getInstance("mate.academy");
+
     public static void main(String[] args) {
+        BookDao bookDao = (BookDao) INJECTOR.getInstance(BookDao.class);
+
+        Optional<Book> foundBook = bookDao.findById(1L);
+        if (foundBook.isPresent()) {
+            System.out.println("Book found: " + foundBook.get());
+        } else {
+            System.out.println("Book not found");
+        }
+        Book book = new Book();
+        book.setId(1L);
+        book.setPrice(new BigDecimal("100.00"));
+        book.setTitle("Alice");
+        bookDao.create(book);
```

Comment:
Creating a book with an already set ID may lead to conflicts since the ID should be auto-generated by the database. You should let the database handle ID generation to avoid potential issues.